docs/build/build-process-management/build-manually-or-with-triggers.md

```diff
@@ -131,6 +131,32 @@ If your commit message includes `[skip ci]` or `[ci skip]`, your workflow will b
 
 If your merge request comment includes `[retry]`, your workflow will be retried.
 
+### Auto Cancel Redundant Pipeline
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/auto-cancel-redundant-pipeline.png' />
```

Comment:
As in previous titles, there should be a description first and then a screenshot.